Scenes/InformationPopup/Popup.tscn

```diff
@@ -0,0 +1,34 @@
+[gd_scene load_steps=2 format=2]
+
+[ext_resource path="res://Scenes/InformationPopup/Popup.gd" type="Script" id=1]
+
+[node name="Popup" type="Control"]
+margin_right = 67.0
+margin_bottom = 24.0
+rect_min_size = Vector2( 12, 12 )
+script = ExtResource( 1 )
+mouse_y_offset = 4.0
+
+[node name="ColorRect" type="ColorRect" parent="."]
+anchor_right = 1.0
+anchor_bottom = 1.0
+grow_vertical = 2
+color = Color( 0.227451, 0.172549, 0.141176, 1 )
+
+[node name="MarginContainer" type="MarginContainer" parent="."]
+anchor_right = 1.0
+anchor_bottom = 1.0
+margin_left = 6.0
+margin_top = 6.0
+margin_right = -6.0
+margin_bottom = -6.0
+grow_vertical = 2
+
+[node name="Label" type="Label" parent="MarginContainer"]
+margin_right = 55.0
+margin_bottom = 14.0
+grow_horizontal = 2
+grow_vertical = 2
+text = "addwa"
```

Comment:
:P
Maybe `default Popup text`

Reply:
Haha, I forgot about that placeholder ;D